core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-tables/contributor-admin-stats-table.component.ts

```diff
@@ -124,61 +124,82 @@ export class ContributorAdminStatsTable implements OnInit {
     return num === 1 ? word : word + 's';
   }
 
-  getCountDisplay(
+  getEditTextForAttribute(
+      editCount: number,
```

Comment:
Try to name this variable so it doesn't start with a verb, or something that could possibly be a verb. For example, numberOfEdits.

Reply:
renamed. Done.